components/datadog/agent/helm.go

```diff
@@ -243,3 +255,22 @@ func buildWindowsHelmValues(installName string, agentImagePath, agentImageTag, _
 		},
 	}
 }
+
+func configureFakeintake(values pulumi.Map, fakeintake *ddfakeintake.ConnectionExporter) {
+	additionalEndpointsEnvVar := pulumi.MapArray{
+		pulumi.Map{
+			"name":  pulumi.String("DD_ADDITIONAL_ENDPOINTS"),
+			"value": pulumi.Sprintf("{\"http://%s\": [\"FAKEAPIKEY\"]}", fakeintake.Host),
```

Comment:
You can use  backquote to avoid escaping `"`

Reply:
Good point !!!